src/shared/index.js

```diff
@@ -32,10 +32,15 @@ async function generate(date, options = {}) {
   }
 
   // Crawler
-  const output = scrapeData(srcs)
```

Comment:
Not sure If this was intended, but it breaks the build!

Reply:
In the future, scraper fix PRs should only touch the scrapers, not core features and logic. That makes them a quick merge that won't cause 45 minutes of head scratching before the release :)